app/_src/gateway/install/linux/amazon-linux.md

```diff
@@ -17,9 +17,51 @@ Kong is licensed under an
 You can install {{site.base_gateway}} by downloading an installation package or using our yum repository.
 
 {% navtabs %}
-{% navtab Package %}
+{% navtab Package (AL2023) %}
+Install {{site.base_gateway}} on Amazon Linux 2023 from the command line.
 
-Install {{site.base_gateway}} on Amazon Linux from the command line.
+1. Download the Kong package:
+
+{% capture download_package %}
+{% navtabs_ee codeblock %}
```

Comment:
❗ Reviewers: These codeblock navtabs don't seem to be displaying and I'm not sure what's wrong with them. Any thoughts for how to fix this?

Reply:
They're working :) they just don't look like tabs. The switch for them is in the upper right corner of the page: 

![Screenshot 2023-05-09 at 4 41 54 PM](https://github.com/Kong/docs.konghq.com/assets/54370747/3c2f0ed0-ab7c-4263-a2a3-f68cccbef170)